src/main/java/org/openstreetmap/atlas/checks/validation/utility/IntersectionUtilities.java

```diff
@@ -0,0 +1,74 @@
+package org.openstreetmap.atlas.checks.validation.utility;
```

Comment:
If you're going the global utility route, it would make more sense to put the utility package under `org.openstreetmap.atlas.checks`